theano/tensor/tests/test_basic.py

```diff
@@ -6026,9 +6026,8 @@ def test_arithmetic_cast(self):
                                     # may be a bug (investigation is currently
                                     # in progress), so in the meantime we just
                                     # mark this test as a known failure.
-                                    raise KnownFailureTest('Known issue with '
-                                            'numpy >= 1.6.x see #761')
-
+                                    raise SkipTest("Not yet implemented")
+                                    # Known issue with numpy >= 1.6.x see #761'
```

Comment:
As far as I know this still happens with 1.9.2.  It's something I've come against in my float16 work.